servers/rendering/renderer_rd/shaders/canvas_uniforms_inc.glsl

```diff
@@ -52,6 +52,8 @@ struct InstanceData {
 #endif
 	vec2 color_texture_pixel_size;
 	uint lights[4];
+	uint instance_uniforms_ofs;
```

Comment:
I haven't taken a look at the whole thing, but just a reminder that this needs to be removed before this PR is finished. We need to avoid bloating the instance struct at all costs

Reply:
Once https://github.com/godotengine/godot/pull/97340 is merged then I'll have the space for it I reckon.

Reply:
Why is that? https://github.com/godotengine/godot/pull/97340 clears up a spot in the instance buffer, but uses that spot for its own needs

Reply:
...oh. I need to find some space then.

Reply:
Here is the space that you can use once #97340 merges:

https://github.com/godotengine/godot/pull/97340/files#diff-a960cbf133befa41681f0d917a05f34b59265036361c79b2c32ace27e0f3a40fR385